test/jdk/javax/swing/JFileChooser/6698013/bug6698013.java

```diff
@@ -45,13 +45,13 @@
     public static void main(String[] args) {
         JFileChooser chooser = new JFileChooser(new VirtualFileSystemView());
         chooser.setCurrentDirectory(root);
-        chooser.showSaveDialog(null);
+        chooser.showOpenDialog(null);
     }
 
     public void init() {
         JFileChooser chooser = new JFileChooser(new VirtualFileSystemView());
         chooser.setCurrentDirectory(root);
-        chooser.showSaveDialog(null);
+        chooser.showOpenDialog(null);
     }
 }
```

Comment:
Shall we eliminate code duplication between main() and init()?

Reply:
init method is part of applet version of test case. It needs to be completely converted to java version. I will do the appropriate changes.

Reply:
> 
> 
> init method is part of applet version of test case. It needs to be completely converted to java version. I will do the appropriate changes.

No, it's not necessary. I meant the opposite. You can create an instance of the test class and call init():
```
SwingUtilities.invokeAndWait(() -> new bug6698013().init());
```
Moreover, Swing components should be created and manipulated on EDT, which does not hold for the current main().

Reply:
I will incorporate the suggested change.